sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Ucfg/UcfgBuilder_Instructions.cs

```diff
@@ -394,5 +454,21 @@ private static void ValidateInstruction(Instruction instruction, string methodId
             }
         }
 
+        private static UCFG CreateUcfgForConstructor(string code, string name)
+        {
+            (var syntaxTree, var semanticModel) = TestHelper.Compile(code, Verifier.SystemWebMvcAssembly);
```

Comment:
[cosmetic] I find the `var (syntaxTree, semanticModel)` construction easier to read.